conf/modules.config

```diff
@@ -22,11 +22,368 @@
 
 params {
     modules {
+        'checksums'{
```

Comment:
Will you update to the latest syntax version? The one that doesn't use params.modules?

Reply:
@mahesh-panchal I am working on this. Hope I will update the pipeline soon.

Reply:
@mahesh-panchal I already update the latest syntax version. Now it is ready for review.